docs/de/docs/index.md

```diff
@@ -22,29 +18,28 @@
 
 ---
 
-**Documentation**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+**Dokumentation**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
 
-**Source Code**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
+**Quellcode**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
 
 ---
 
-FastAPI is a modern, fast (high-performance), web framework for building APIs with Python 3.6+ based on standard Python type hints.
+FastAPI ist ein modernes, schnelles (High-Performance) Web-Framework zur Erstellung von APIs mit Python 3.6+ auf Basis von Python type hints.
+Die Haupteigenschaften sind:
 
-The key features are:
+* **Schnell**: Sehr hohe Leistung, auf dem Niveau von **NodeJS** und **Go** (Dank an Starlette und Pydantic) [Eines der schnellsten verfügbaren Python-Frameworks](#performance).
```

Comment:
I would translate `thanks to [...]` only with `dank [...]`

Reply:
Yes would remove the "an"